test/fixtures/scenarios/logstashFunctional/makelogsIndexDefinition.js

```diff
@@ -47,7 +43,6 @@ module.exports = {
         },
         'id': {
           'type': 'integer',
-          'index': 'not_analyzed',
```

Comment:
Maybe these should be set to `index: true` to retain the same meaning as before?